Gems/Atom/RPI/Code/Source/RPI.Public/RPISystem.cpp

```diff
@@ -80,21 +80,6 @@ namespace AZ
 
         void RPISystem::Initialize(const RPISystemDescriptor& rpiSystemDescriptor)
         {
```

Comment:
Make sure that if link mode is not initiated and someone passed in -openxr=enable it doesn't crash and just gracefully handles disabling xrsystem with an AZ_error message.